components/Shipping/exceptions.rst

```diff
@@ -0,0 +1,38 @@
+Exceptions
+==========
+
+.. note::
+    This exceptions extends the `PHP InvalidArgumentException`_.
```

Comment:
I'd prefer having this note at the end of file and written somewhat like: 
`All of above exceptions blablabla`
because the info when a given exception is thrown is far more important than what it extends.